### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by Tim-Jackins**
Just some general stuff to make the repo look more official and add clarity. I'd like to be able to follow standard-readme style but there are certain sections I can't add so I can't really argue for it but I'd love to hear your opinion.

### CODE REVIEW

**Comment by Tim-Jackins** [comment]
I went ahead and removed some of the public aspects to avoid confusion.

File: README.md [hunk 1]
```diff
@@ -6,32 +6,17 @@
 
 >A set of tutorials and challenges to ready students for collaboration via Git and GitHub
 
-These tutorials are designed to be added into a [Canvas course](https://www.canvaslms.com/). This is why they don't conform to CommonMark Spec.
+These tutorials are for **Charlottesville High School Students** and are designed to be added into a [Canvas course](https://www.canvaslms.com/).
```
[the remark this answers is not in the document]
Comment (jonahweissman) [reply README.md]: I'm not sure why GitHub thinks this is outdated; it still needs to be addresssed

**Comment by Tim-Jackins** [comment]
Do you have anymore suggestions?

**Comment by jonahweissman** [comment]
Yeah, I have comments on the section that says

> These tutorials are for **Charlottesville High School Students** and are designed to be added into a [Canvas course](https://www.canvaslms.com/). 

but GitHub is saying they're outdated. Here are my thoughts (some are new):

* I think you should say how often you'll update Canvas to match this repo (could be just whenever a PR is merged). 
* Unless I'm missing something, you should say "intended to be added to Canvas" not "designed". 
* No need for a link to Canvas, CHS students know what it is.

**Pull request merged**